src/posts/paper/13-q-and-a.ts

```diff
@@ -179,6 +179,48 @@ However, we have retracted this method for the following reasons:
+risky activities (for instance, the Moderna vaccine confers a 1/5x multiplier for incoming microCOVIDs, so a vaccinated person who does 5x more risky behavior
+after being vaccinated would be just as many microCOVIDs as before).
+
+There are two questions that are important for understanding vaccines:
+
+1. How much less likely is it for a vaccinated individual to catch COVID? This determines the reduction in microCOVIDs a vaccinated individual receives from a given action.
+2. If a vaccinated individual contracts COVID, how much less (or more) likely is this to result in negative consequences (hospitalizaton or death). This determines how much a person can increase their microCOVID budget for being vaccinated (on top of being able to do more activities from part 1).
+
+### 1. How much less likely is it for vaccinated individuals to catch COVID? (Reduction in microCOVIDs)
+
+We do not know for sure how likely it is for a vaccinated individual to catch to COVID. Vaccinated individuals still have a chance of becoming sick (even if they are not seriously ill), and may be asymptomatic carriers of the virus, capable of infecting others without having symptoms themselves. None of the vaccine studies have directly addressed what the chances of transmission from a vaccinated individual are. We attempt to make educated guesses by pulling in other sources. For our derivation of these numbers, see the [Research Sources](/paper/14-research-sources#vaccines).
+
+|             | microCOVID multiplier 7 days after 1st dose | microCOVID multiplier 14 days after 2nd dose |
+| ----------- | ------------------------------------------- | -------------------------------------------- |
+| AstraZeneca | 0.56                                        | 0.4                                          |
+| Moderna     | 0.56                                        | 0.2 [^moderna]                               |
+| Pfizer [^pfizer] | 0.56                                   | 0.2                                          |
+
+For perspective, the first dose of either vaccine confers equivalent protection as wearing a good quality cloth mask all day, every day. Completing the 2nd dose of AstraZeneca’s vaccine confers more protection than wearing a surgical mask 24/7, and Pfizer or Moderna’s vaccine is as protective as wearing a KN95 mask all the time.
+
+While the vaccines are nowhere near making you invulnerable to COVID, remember that these vaccines are absolutely game changing - universal vaccination would reduce *everyone’s* chance of catching COVID by 50% or more. This will (likely) reduce the rate of community spread to the point that every week has fewer cases than the last (R<1). The end of the pandemic is in sight. 
+
+### 2. If a vaccinated individual contracts COVID, how much less (or more) likely is this to result in negative consequences?
```

Comment:
Add to the end of the header: " (increased budget)"

This makes it symmetrical to the header above.